### DISCUSSION TITLE
Using fastMRI data in public notebooks

### DISCUSSION THREAD

**Discussion by zaccharieramzi** [facebookresearch/fastMRI#197] [Q&A]
Hello,

I am a big fan of what the fastMRI team has done over the last years, so this is for me the opportunity to thank you all for that once more.
I would like to use some volumes from the fastMRI dataset in public notebooks, typically to showcase how to use the networks I have developed in particular for the fastMRI challenge of 2020. 
Could I do that? I would of course put all due credit in the said notebooks.

I am not sure exactly how this use case fits in the [sharing agreement](https://fastmri.med.nyu.edu/) I have agreed to.
I had actually sent an email in March 2020 about a related topic to `fastmri@med.nyu.edu` but did not get a response.

Thanks in advance for your answer

EDIT
-----
I would like to specify that I could do with just some slices and not whole volumes.
I don't know if that changes anything from your point of view.

**Comment by mmuckley** [comment]
Hello @zaccharieramzi, unfortunately this is not something I can address myself and you will have to contact NYU directly at fastmri@med.nyu.edu - they have the sole legal authority over the data.

**Comment by zaccharieramzi** [comment]
Sure I can do that! Is there a way to maybe make them aware of this question via other means because as I said last time I tried to contact them I did not get a response?

**Reply by mmuckley** [reply]
Okay I'll ping them too :).

**Comment by zaccharieramzi** [comment]
Thanks !! Writing the email right now
